modVM/src/org/aion/vm/KernelTransactionContext.java

```diff
@@ -160,7 +169,7 @@ public Address getDestinationAddress() {
     @Override
     public Address getContractAddress() {
         byte[] rawBytes = this.transaction.getContractAddress().toBytes();
-        rawBytes[0] = 0x0f;
+        rawBytes[0] = NodeEnvironment.CONTRACT_PREFIX;
```

Comment:
These are in fact different constants, it just happens that their current values are the same. The `AVM_CREATE_CODE` is the transaction type that the executor uses to determine which VM to use. But this is the contract prefix, the expected first byte of any AVM contract address, which is used for calls and for determining whether a VM is performing a cross-vm call. 

This might also be represented in a different way in the future, so good to not confuse them.